internal/util/fileop_test.go

```diff
@@ -358,6 +414,185 @@ func TestFileOpCopyBase(test *testing.T) {
 	}
 }
 
+func TestFileOpCopyGlob(test *testing.T) {
+	testCases := []struct {
+		source          string
+		dest            string
+		expectedSources []string
+		expectedDests   []string
```

Comment:
I could be missing something, but I'm not sure why there are two here.
Are the expected sources and dests treated differently, or can we just combine them?

If this is for symmetry with the move test, maybe just have expectedExists and expectedNotExists slices.